examples/editor/src/App.tsx

```diff
@@ -1,26 +1,150 @@
 // import logo from './logo.svg'
 import "@blocknote/core/style.css";
-import { BlockNoteView, useBlockNote } from "@blocknote/react";
+import {
+  BlockNoteView,
+  getDefaultReactSlashMenuItems,
+  useBlockNote,
+  Theme,
+  defaultTheme,
+} from "@blocknote/react";
 import styles from "./App.module.css";
+import { BlockSchema, defaultBlockSchema } from "@blocknote/core";
+import { Image, insertImage } from "../../../tests/utils/customblocks/Image";
+import {
+  insertReactImage,
+  ReactImage,
+} from "../../../tests/utils/customblocks/ReactImage";
 
 type WindowWithProseMirror = Window & typeof globalThis & { ProseMirror: any };
 
+const redTheme: Partial<Theme> = {
```

Comment:
I think adding dark and light modes as part of a theme creates a more complex API, and also more work (if you're creating a "red" theme, you'll need to implement both dark and light mode for it. In practice, what I think will happen, is that people will just insert duplicates or dummy modes for one).

I think what we should do instead is making BlockNoteLightTheme and BlockNoteDarkTheme default themes that we expose.

The `theme` prop on the editor could then take: `Theme | { dark: Theme, light: Theme}` and defaults to `{dark: BlockNoteDarkTheme, light: BlockNoteLightTheme}`. 

If a dark/light config is passed, we should auto-switch to the right theme based on browser settings

Reply:
Yep I agree with this. How about the `useDarkMode` prop, do you think we should still keep it and have it default to the user's browser/OS setting? Or remove it entirely and just decide which to use based on browser/OS setting?

Reply:
I think we can remove it. If someone wants to manually control it, they can just pass the light or darktheme directly as “theme”